src/js/hyperschedule.js

```diff
@@ -1669,21 +1669,66 @@ function downloadPDF()
           course.courseName,
           width - 12,
         );
+        const courseLocationLines = pdf.splitTextToSize(
+          slot.location,
+          width - 12,
+        );
+        const courseInstructorsLines = pdf.splitTextToSize(
+          course.faculty.map(fullName => fullName.split(",")[0]).join(", "),
+          width - 12
+        );
 
+        // Attributes to be shown are calculated based on space available
+        // order of preference: Code, location, name, professor
+        // order of display: code, name, professor, location
+        const maxTextLength = (yEnd - yStart) - 2 * pdf.getLineHeight();
```

Comment:
Why are we subtracting `2 * pdf.getLineHeight()` here?

Reply:
Noted in comment